newIDE/app/src/Profile/CurrentUsageDisplayer.js

```diff
@@ -48,14 +48,26 @@ const CurrentUsageDisplayer = ({
   const usageRatio = `${usedBuilds}/${quota.max}`;
   const remainingMultipleMessage = (
     <Trans>
-      You have {remainingBuilds} builds remaining (you have used {usageRatio} in
-      the last 24h).
+      You have <b>{remainingBuilds}</b> builds remaining — you have used
+      {usageRatio} in
+      {quota.period === '30days' ? (
+        <Trans>the past 30 days</Trans>
+      ) : (
+        <Trans>the past 24h</Trans>
+      )}
+      .
     </Trans>
   );
   const remainingSingleMessage = (
     <Trans>
-      You have {remainingBuilds} build remaining (you have used {usageRatio} in
-      the last 24h).
+      You have <b>{remainingBuilds}</b> build remaining — you have used
```

Comment:
is there a risk this whole sentence is hard to translate in certains languages because of all those different bits?

Reply:
This should create a "You have {0} build remaining - you have used {1} in {2}" so... it's correct but a bit of a pain to translate.

Reply:
I will make instead 2 different sentences